packages/block-editor/src/components/block-list/block.js

```diff
@@ -737,6 +775,9 @@ const applyWithDispatch = withDispatch( ( dispatch, ownProps, { select } ) => {
 		toggleSelection( selectionEnabled ) {
 			toggleSelection( selectionEnabled );
 		},
+		enableNavigationMode() {
+			toggleNavigationMode( true );
```

Comment:
Minor: "Toggle" to me means you inverse a boolean value, depending on the current value. Here `setNavigationMode` or something similar might make more sense?

Reply:
makes sense.